metabase/metabase
# Handle text tag differences when selecting a custom values source

Epic https://github.com/metabase/metabase/issues/26898

Changes:
- Remove `From connected fields` option for text tags and pre-select `From another question...`
- Make `Input box` the default option for text tags
- Make `ParameterFieldWidget` be used when the parameter has either `dropdown` or `search` type

How to test:
- New -> SQL Query
- `SELECT {{tag}}`
- The default option in the sidebar should be `Input box`
- `Dropdown list` -> `Edit`
- There should be no `From connected fields`

<img width="1135" alt="Screenshot 2023-01-18 at 16 02 12" src="https://user-images.githubusercontent.com/8542534/213191792-67858f12-1faa-47e8-b37a-d2bff83fbee6.png">
<img width="362" alt="Screenshot 2023-01-18 at 16 02 17" src="https://user-images.githubusercontent.com/8542534/213191809-06a25599-a4ad-4f33-95a0-b7fe82a4edca.png">
